subsys/bluetooth/host/conn.c

```diff
@@ -2374,16 +2374,55 @@ static int bt_conn_get_tx_power_level(struct bt_conn *conn, uint8_t type,
 	return 0;
 }
 
+#if defined(CONFIG_BT_TRANSMIT_POWER_CONTROL)
+int bt_conn_le_enhanced_get_tx_power_level(struct bt_conn *conn,
+				  struct bt_conn_le_tx_power *tx_power)
```

Comment:
Do we really need this #ifdef? If there are no calls to the function during the build, the linker should just strip it out from the final image, shouldn't it? Additionally, dropping the #ifdef should let you use `if (IS_ENABLED())` when calling the function, instead of an #ifdef.